runelite-client/src/main/java/net/runelite/client/plugins/mining/Rock.java

```diff
@@ -90,7 +90,11 @@ Duration getRespawnTime(int region)
 	ORE_VEIN(Duration.ofSeconds(MiningOverlay.ORE_VEIN_MAX_RESPAWN_TIME), 150),
 	AMETHYST(Duration.ofSeconds(75), 120),
 	ASH_VEIN(Duration.ofSeconds(30), 0, ASH_PILE),
-	GEM_ROCK(Duration.ofMinutes(1), 0, ROCKS_11380, ROCKS_11381);
+	GEM_ROCK(Duration.ofMinutes(1), 0, ROCKS_11380, ROCKS_11381),
+	URT_SALT(Duration.ofMillis(5400), 0, ROCKS_33254),
+	EFH_SALT(Duration.ofMillis(5400), 0, ROCKS_33255),
+	TE_SALT(Duration.ofMillis(5400), 0, ROCKS_33256),
+	BASALT(Duration.ofMillis(5400), 0, ROCKS_33257);
```

Comment:
Since #10773 has been merged, these can be changed to use `Duration.of(9, GAME_TICKS)` so long as `RSTimeUnit.GAME_TICKS` is imported.

Reply:
Changed, let me know if it is also better to update the rest of the enum to match with the GAME_TICKS also.